README.md

```diff
@@ -320,19 +320,20 @@ With Git-Flow, branches are categorised into
 * One or several **feature** branches.
 * One or several **hotfix** branches.
 
-The code on the **master** branch (often called `master` or `release`) is
+The code on the **master** branch (often called `master`) is
```

Comment:
Github calls the default branch `main` now, so that will probably become common in the future.

Reply:
Yes we should update that

Reply:
It is somewhat confusing with this statement. The text below is explaining the meaning of using a `release` branch, but here it says the  `master` branch and the `release` branch are often identical.

Reply:
I find more clear the first version The code on the **master** branch (often called `master` or `release`) is...
since the coming sentences are explaining the release process.